gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/infra/crud_service/event/EventCrudServiceLegacyWrapper.java

```diff
@@ -42,14 +41,29 @@ public Event createEvent(
         Object content,
         Map<Event.EventProperties, String> properties
     ) {
-        return EventAdapter.INSTANCE.fromEntity(
-            eventService.createEvent(
-                new ExecutionContext(organizationId, environmentId),
-                environmentIds,
-                eventType,
-                content,
-                EventAdapter.INSTANCE.toStringEventPropertiesMap(properties)
-            )
+        return this.createEvent(organizationId, environmentId, environmentIds, eventType, content, properties, false);
+    }
+
+    @Override
+    public Event createEvent(
```

Comment:
instead of an overload would it be preferable to have a meaningful function name ? 
like `createEventAndMarkAsLatest`
when the service will grow the risk is to have method calling each other with a list of flag parameters